global_constants.py

```diff
@@ -2,17 +2,34 @@
 from pygame.locals import *
 from sys import intern
 
-# SCREEN_WIDTH = 1280
-# SCREEN_HEIGHT = 720
 SCREEN_WIDTH = 1600
 SCREEN_HEIGHT = 900
 SCREEN_CENTER = (SCREEN_WIDTH/2, SCREEN_HEIGHT/2)
 FPS = 60
 
+MAP_EMPTY = '.'
+MAP_FLOOR = '1'
+MAP_7 = '7'
+MAP_F = 'F'
+MAP_J = 'J'
+MAP_L = 'L'
+
+PROJECTION_TILE_X_OFFSET = 50  # tile size in px, basically
+PROJECTION_TILE_Y_OFFSET = 50  # tile size in px (actual floor portion only), basically
+PROJECTION_GROUND_LEVEL_ORIGIN = 0, SCREEN_HEIGHT//2  # the place on the screen where the top left ground level should be. revisit this later. possibly name this something like 'HORIZON_TOP_LEFT'
+
+PLAYER_WIDTH = 50
+PLAYER_HEIGHT = 100
```

Comment:
I've been going back and forth on whether some of these constants belong in one centralized place (which is arguably the point of this file), or whether they belong in their respective class files. Many of them started off in their own class files, but over time we see that other classes need to refer to them, so it's easier if they live in one centralized place.

I would suggest a convention of having the first word of the constant match the class that the value generally belongs to. We should also group these together in the same places within this file.